packages/playwright-test/index.mjs

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import * as playwright from './index.js';
+import playwright from './index.js';
```

Comment:
When node imports a commonjs module, it sets the entire module as the default. There is no way to destructure values from a commonjs module, and if you do `* as foo` you get an object like `{default: 'foo'}`.